scheduler_simulator.py

```diff
@@ -193,6 +193,7 @@ class Path:
     name: str
     interface: Interface
     cc: CongestionController
+    priority: int = 0
```

Comment:
Could you move it one below the comment and discuss it in section [Packet schedulers](https://github.com/obonaventure/draft-schedulers/blob/master/draft-schedulers.mkd#packet-schedulers-schedulers) ?

Reply:
I just saw your comment and I send a fix in between when I looked at the order (highest priority first → we need to reverse the list)